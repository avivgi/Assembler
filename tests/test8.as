.define len=2
 LIST:   .data   6, -9, len
MAIN:           mov     r3, LIST[2]