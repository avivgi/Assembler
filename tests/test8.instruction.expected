Address: 100	value: 0	 word: 00000000111000
Address: 101	value: 3	 word: 00000001100000
Address: 102	value: 104	 word: 00000110100010
Address: 103	value: 2	 word: 00000000001000
